unlock-protocol.com/blog/blog.index

```diff
@@ -1 +1 @@
"hello-optimistic-unlocking"},{"title":"Ways to Unlock the Web","subTitle":"We’re building an open protocol to allow anyone on the web to earn money from their work. The possibilities are endless. Here are some of them.","authorName":"Ben Werdmuller","publishDate":"March 20, 2019","description":"We’re building an open protocol to allow anyone on the web to earn money from their work. The possibilities are endless. Here are some of them.","image":"/static/images/blog/waystounlocktheweb/hero.jpg","slug":"ways-to-unlock-the-web"},{"title":"What is Unlock?","subTitle":"Or, why we’re building a business model as an open protocol.","authorName":"Julien Genestoux","publishDate":"February 26, 2019","description":"Why we’re building a business model as an open protocol.","image":"/static/images/blog/whatisunlock.jpg","slug":"what-is-unlock"},{"title":"Announcing Unlock Version Zero","subTitle":"We’re changing the business model for the web. It starts here, today.","authorName":"Ben Werdmuller","publishDate":"January 25, 2019","description":"We’re changing the business model for the web. It starts here, today.","image":"/static/images/blog/announcingv0/paywalldemo.png","slug":"announcing-v0"},{"title":"The End of the Ad-Supported Web","subTitle":"What if we got the business model of the web wrong?","authorName":"Julien Genestoux","publishDate":"December 13, 2018","description":"What if we got the business model of the web wrong?","image":"/static/images/blog/adsupportedweb/1984.jpg","slug":"ad-supported-web"},{"title":"To Save Freedom of Speech, We Must Decentralize the Web","subTitle":"In a world where corporations control our conversations, removing middlemen is more important than ever.","authorName":"Ben Werdmuller","publishDate":"October 15, 2018","description":"In a world where corporations control our conversations, removing middlemen is more important than ever.","image":"/static/images/blog/freedomofspeech/thisisforeveryone.jpg","slug":"freedom-of-speech"},{"title":"Unlocking Some Exciting News","subTitle":"We’re growing our talented and mission-driven team. Come join us!","authorName":"Julien Genestoux","publishDate":"July 24, 2018","description":"We’re growing our talented and mission-driven team. Come join us!","slug":"exciting-news"},{"title":"Setting Unlock’s demo free","subTitle":"Today we’re sharing the very first demo of the Unlock Protocol.","authorName":"Julien Genestoux","publishDate":"June 10, 2018","description":"Today we’re sharing the very first demo of the Unlock Protocol.","slug":"setting-unlocks-demo-free"},{"title":"It’s Time to Unlock The Web","subTitle":"The web needs a better business model — and we believe the technology is finally here to do it.","authorName":"Julien Genestoux","publishDate":"April 27, 2018","description":"The web needs a better business model — and we believe the technology is finally here to do it.","image":"/static/images/blog/itstimetounlocktheweb/unlocklegacy.jpg","slug":"its-time-to-unlock-the-web"}]}
```

Comment:
I created this branch off a fresh pull from master a few minutes before I opened the PR. Just tried rebasing, and there were no changes. Not sure why it's different, but running the app caused that to change.

Reply:
weird. Must be using the button, which causes a regenerate?

Reply:
Possibly. Now that I look at it, it's possible a blog post was changed and committed without building the blog index, so since I happened to run the app my PR is the one that picked up the change. I can split it out, since there is actually a real different in the blog index (the subtitle field, at least)